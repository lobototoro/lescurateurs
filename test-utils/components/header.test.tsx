import { render } from '@testing-library/react';
import { expect, it } from 'vitest';

import Header from '@/app/components/header';

it('Should render correctly', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});