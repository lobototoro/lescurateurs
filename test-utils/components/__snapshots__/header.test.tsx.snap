// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render correctly 1`] = `
<DocumentFragment>
  <header
    class="_header-title_f11b8f container"
  >
    <div>
      <h1
        class="undefined article-title_extra-large article-title_black"
      >
        LES CURATEURS
      </h1>
    </div>
  </header>
</DocumentFragment>
`;
